src/identity.js

```diff
@@ -721,6 +730,45 @@ export class Identity extends EventEmitter {
         return null;
     }
 
+
+    /**
+     * Starts sms/otp/password login flow for logged in users. Flow results with user access
+     * token with intended methods in amr field (make sure to verify token amr).
```

Comment:
The point "make surt to verify token amr" should be emphasised more (the entire security of the re-authentication depends on it). And maybe some details about how to verify it (that it's a claim in the ID token and how to check that).